n6001/README.md

```diff
@@ -50,13 +50,13 @@ and updates the project qsf files appropriately.
 
 Need to set **OPAE_PLATFORM_ROOT** to point to ofs-dev/work_ofs_ac_base_adp-A1/pr_build_template in FIM build area.
 
-Need to set **OFS_OCL_SHIM_ROOT** to point to opencl-bsp/n6001.
+Need to set **OFS_OCL_SHIM_ROOT** to point to opencl-asp/n6001.
```

Comment:
its should be oneapi-asp

Reply:
we have also changed OFS_OCL_SHIM_ROOT to OFS_ASP_ROOT. I think most of change sin this PR might already be in master now

Reply:
good catch

Reply:
found it in d5005 fixing that as well

Reply:
> we have also changed OFS_OCL_SHIM_ROOT to OFS_ASP_ROOT. I think most of change sin this PR might already be in master now

Synced to main and then saw these!